### DISCUSSION TITLE
How to adjust CEF configuration of Lively wallpaper

### DISCUSSION THREAD

**Discussion by matthiasstorr** [rocksdanister/lively#697] [Q&A] [answered]
Hey,
I want to make a widget like wallpaper on my desktop with the use of html.
Because of this i want to embed websites via the iframe tag. 

Now i struggle in the implementation. I want to request the mobile version of the website but there is no mobile specific url available. This reults in two questions:
1.) I want to edit the http request header to send an mobile user agent but how can i change this setting in lively wallpaper?
2.) Also i want to allow cookies so the embeded websites are displayed properly - how can i enable cookies in lively wallpaper?

I hope someone can help me with my problems or can give me alternatives to get to my goal.

**Comment by rocksdanister** [comment, marked as answer]
Currently there is no way to do this unless you compile a custom version of the browser lively is using.

**Reply by matthiasstorr** [reply]
Thanks for your fast reply. But the source code belongs to you and is not available to third parties - if i am informed correctly.
My Opinion is that this should be the case also in the future. Thus, my question if you could probably compile a custom version tailored to my use case?

**Reply by rocksdanister** [reply]
>But the source code belongs to you and is not available to third parties - if i am informed correctly.

No

https://github.com/rocksdanister/lively-cef/blob/6c40aa5953532a0061129383796064ceb0284466/src/cefsharptest/Form1.cs#L518

Just needs `CefSettings.UserAgent` flag set.

>Thus, my question if you could probably compile a custom version tailored to my use case?

I think its better to have optional json config file with some useful options anyone can use.. will try to do it next week.

**Reply by rocksdanister** [reply]
Issue https://github.com/rocksdanister/lively/issues/706

**Reply by matthiasstorr** [reply]
haha, i didnt saw it - thanks a lot
